testutil/test_helper.go

```diff
@@ -127,19 +127,19 @@ func (h *TestHelper) CombinedOutput(name string, arg ...string) (string, error)
 	return string(bytes), nil
 }
 
-// ConduitRun executes a conduit command appended with the --conduit-namespace
+// LinkerdRun executes a linkerd command appended with the --linkerd-namespace
 // flag.
-func (h *TestHelper) ConduitRun(arg ...string) (string, error) {
-	withNamespace := append(arg, "--conduit-namespace", h.namespace)
-	return h.CombinedOutput(h.conduit, withNamespace...)
+func (h *TestHelper) LinkerdRun(arg ...string) (string, error) {
```

Comment:
tioli: I noticed you've been renaming some functions to remove the specific product name from them. This could be `CliRun`/`CliRunStream` though maybe it is good that it's the name of the specific command being run.

Reply:
Good observation. I think it's worth making an exception for these commands, since they're actually running the `linkerd` binary.